Java/core/src/main/java/com/amazon/randomcutforest/imputation/ImputeVisitor.java

```diff
@@ -94,14 +131,8 @@ public ImputeVisitor(double[] queryPoint, int numberOfMissingIndices, int[] miss
         this.missing = Arrays.copyOf(original.missing, length);
         this.liftedPoint = Arrays.copyOf(original.liftedPoint, original.liftedPoint.length);
         this.liftedMissing = Arrays.copyOf(original.liftedMissing, original.liftedPoint.length);
-        rank = 10.0;
-    }
-
-    /**
-     * @return the rank of the imputed point in this visitor.
-     */
-    public double getRank() {
-        return rank;
+        anomalyRank = Double.MAX_VALUE / 2;
```

Comment:
Why doesn't this use `DEFAULT_INIT_VALUE`? If the point is that we want to use a value that's strictly larger than in the visitor we're copying, won't that fail if the visitor has to split twice (i.e., because there are 2 missing values in the input point)?

Reply:
Oversight. Fixed.